src/corecel/math/Algorithms.hh

```diff
@@ -648,8 +648,9 @@ CELER_WRAP_MATH_FLOAT_DBL_1(, sinpi)
 CELER_WRAP_MATH_FLOAT_DBL_1(, cospi)
 CELER_WRAP_MATH_FLOAT_DBL_PTR_2(, sincospi)
 CELER_WRAP_MATH_FLOAT_DBL_PTR_2(, sincos)
-#elif __APPLE__ && !defined(__CLING__)
-// Apple defines __sinpi, __sinpif, __sincospi, ...
+#elif (__MAC_OS_X_VERSION_MIN_REQUIRED > 1090) \
```

Comment:
it looks odd to use a `__` macro, isn't there a "public" version we can use here?

Reply:
Alas no, I tried. They're defined in [Availability.h](https://opensource.apple.com/source/xnu/xnu-7195.81.3/EXTERNAL_HEADERS/Availability.h.auto.html) and it says they're "for use in OS header files" but at first reading I didn't notice the "OS" — so maybe we're not allowed to use them ☹️ the fallback still works though...

The only other thing I could do is add a cmake compiler test instead but that seems like overkill.

Reply:
> The only other thing I could do is add a cmake compiler test instead but that seems like overkill.

it is however the 'right' thing to do ... and who even catch other compiler/os that supports it.

Reply:
Thinking about it, `__APPLE__`  and `__CLING__` are both underscores too...